test/e2e/e2eutil.go

```diff
@@ -194,3 +201,99 @@ func NewTestTLSConfig(optional bool) mdbv1.TLS {
 		},
 	}
 }
+
+func ensureObject(ctx *f.Context, t *testing.T, key types.NamespacedName, obj runtime.Object) {
+	msg := ""
+	if key.Namespace != "" && key.Namespace != "default" {
+		msg = fmt.Sprintf("%s %s/%s", reflect.TypeOf(obj), key.Namespace, key.Name)
+	} else {
+		msg = fmt.Sprintf("%s %s", reflect.TypeOf(obj), key.Name)
+	}
+
+	err := f.Global.Client.Get(context.TODO(), key, obj)
+	if err != nil && !apiErrors.IsNotFound(err) {
+		t.Fatal(err)
+	} else if err == nil {
+		fmt.Printf("%s already exists!\n", msg)
+		err = f.Global.Client.Update(context.TODO(), obj)
+		if err != nil {
+			t.Fatal(err)
+		}
+	} else {
+		err = f.Global.Client.Create(context.TODO(), obj, &f.CleanupOptions{TestContext: ctx})
+		if err != nil {
+			t.Fatal(err)
+		}
+	}
+}
+
+// EnsureNamespace checks that the given namespace exists and creates it if not
```

Comment:
As a note, we don't have it wrote anywhere for the Community Operator, but in general go documentation comment for function should always end with a full stop.

Reply:
Sounds good! I tried to update all the comments I added to follow this convention, but please lmk if I missed any.